js/src/dapps/registry/Accounts/accounts.js

```diff
@@ -61,20 +65,27 @@ export default class Accounts extends Component {
   }
 
   renderAccount = (account) => {
-    const { all, selected } = this.props;
+    const { selected } = this.props;
     const isSelected = selected && selected.address === account.address;
 
     return (
       <MenuItem
         key={ account.address } value={ account.address }
         checked={ isSelected } insetChildren={ !isSelected }
```

Comment:
Not related, however not sure why this stayed this way - everywhere we have an attribute per line